### ISSUE TITLE
how to run

### ISSUE THREAD

**Issue by suke27** [ai-tor/PyTorch-SRGAN#5]
hi, can you tell how to run after train done, seems no test program

**Comment by suke27** [comment]
i make test.py by myself, but it seems the image is not right. the out image is garbled. can you share me the pretrain model

**Comment by ai-tor** [comment]
Hi! I've improved training and added a test script. Take a look at it

**Comment by suke27** [comment]
yes, seems your test script is for cifar100, image size is 15 fixed, do you test using real image

**Comment by aitorzip** [comment]
True, I replaced by padding 4 a few days ago.
